openstack/keystone/alerts/kubernetes/keystone.alerts

```diff
@@ -16,3 +16,18 @@ groups:
     annotations:
       description: 'The keystone ingress API endpoint is throwing 5xx for 5 min. Check if the keystone-api k8s pods are OK.'
       summary: 'Keystone API ingress endpoint is not healthy'
+
+  - alert: OpenstackKeystoneDatabaseSize
+    expr: predict_linear(pvc_usage{persistentvolumeclaim="db-keystone-pvclaim"}[1h], 24 * 3600) * 100 >= 90
```

Comment:
Prediction is based on the data of the last hour and forecasting for the next 24 hours.

I would like to change two things:
1. increase the range of base data, e.g. to tell what might happen in the next 24 hours look back what happened in the last 6 hours to smoothen extreme fluctuations
2. increase the time to react: currently we would be warned 1 day upfront (and I bet this will happen on the weekend). Better warn 1 week ahead instead

Combined I imagine something like
`predict_linear(pvc_usage{persistentvolumeclaim="db-keystone-pvclaim"}[1d], 7 * 24 * 3600) * 100 > 90`

"If it grows for 7 days like it has grown for the past day, the pvc would be used more than 90 percent"

And maybe add a guard to prevent false positives:
`(pvc_usage{persistentvolumeclaim="db-keystone-pvclaim"} * 100 > 70) and predict_linear...`
"only warn if currently already 70 percent is used"